cli-plugins/apps/create-workload.hbs.md

```diff
@@ -127,9 +116,55 @@ tanzu apps workload create petclinic-image --param-yaml maven=$"artifactId:hello
 tanzu apps workload create petclinic-image --param-yaml maven="{"artifactId":"hello-world", "type": "jar", "version": "0.0.1", "groupId": "carto.run"}"
 ```
 
+## <a id='yaml-files'></a> Working with YAML files
+
+In many cases, you can manage workload life cycles through CLI commands.
+However, you might find cases where you want to manage a workload by using a `yaml` file.
+The Apps CLI plug-in supports using `yaml` files.
+
+The plug-in is designed to manage one workload at a time.
+When you manage a workload using a `yaml` file, that file must contain a single workload definition.
```

Comment:
When you manage a workload using a `yaml` file, that file **must contain a single workload definition**.